apps/google-analytics-4/frontend/src/components/config-screen/map-account-property/MapAccountPropertySection.tsx

```diff
@@ -77,39 +78,47 @@ export default function MapAccountPropertySection(props: Props) {
       <div>
         <Subheading marginBottom="spacingXs">Google Analytics 4 property</Subheading>
         <Paragraph marginBottom="none">
-          Note: Only <em>Google Analytics 4</em> properties are listed.{' '}
-          <em>Google Universal Analytics</em> properties are not supported. See{' '}
-          <TextLink
-            href="https://support.google.com/analytics/answer/10759417"
-            target="_blank"
-            icon={<ExternalLinkIcon />}
-            alignIcon="end">
-            "Make the switch to Google Analytics 4"
-          </TextLink>{' '}
-          for more information.
+          In the dropdown below, select the Google Analytics 4 property that's connected to the
+          website where content in this Contentful space (environment) appears.
```

Comment:
Recommend `website where this content appears.` to reduce the complexity here. It's difficult to read and understand. We need to dumb things down